app/controllers/admins_controller.rb

```diff
@@ -55,6 +55,23 @@ def update
     redirect_to admins_url, notice: "Admin was successfully updated."
   end
 
+  def resend_invitation
```

Comment:
Should we write a controller spec for this?

It'll be a little awkward to assert since everything is just a redirect with a `notice` or `alert` without any real status codes but at least the branching will get tested over?